SPECS/shadow-utils/shadow-utils.spec

```diff
@@ -178,9 +176,6 @@ chmod 000 %{_sysconfdir}/shadow
 %{_libdir}/libsubid.so
 
 %changelog
-* Tue May 02 2023 CBL-Mariner Servicing Account <cblmargh@microsoft.com> - 4.9-12
```

Comment:
why is this shadow-utils patch being reverted?

Reply:
This patch was causing problems in the build some time ago so I had reverted the change to help see my builds through in the pipelines prior to getting onto main/2.0. This commit should likely be taken out, given the fix was applied to 2.0/main a couple weeks ago here: https://github.com/microsoft/CBL-Mariner/commit/39e584023c93299673fff9b8ff6b3b7e3978926b